github/repository/models.py

```diff
@@ -1,12 +1,14 @@
 from django.db import models
 from wiki.models import Wiki
+from django.contrib.auth.models import User
 from django.utils.translation import gettext_lazy as _
 
 
 class Repository(models.Model):
     name = models.CharField(max_length=50)
     creationDate = models.DateTimeField(auto_now_add=True)
     wiki = models.OneToOneField(Wiki, on_delete=models.CASCADE)
+    user=models.ForeignKey(User, verbose_name='user', on_delete=models.CASCADE, default=None)
```

Comment:
Ovo je veza user-a koji je napravio repository? Zasto nema veze sa userima koji su clanovi repozitorijuma?

Reply:
To ce biti dodato u nekom sledecem feature-u kada se bude napravilo dodavanje clanova na repozitorijum